public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```

Comment:
these need the same names as what's in manifest.json

Also can we call it `manifest.webmanifest`? it's what the [spec](https://developer.mozilla.org/en-US/docs/Web/Manifest) says